src/tls.c

```diff
@@ -4697,6 +4700,33 @@ int TLSX_ValidateSupportedCurves(WOLFSSL* ssl, byte first, byte second) {
             }
         }
 
+        /* REVIEWER: ECDHE_PSK_WITH_AES_128_CBC_SHA256 is not present in this
+         *           function. So I'm guessing this is not neccessary? Or...do
+         *           we need to add a handler for
+         *           ECDHE_PSK_WITH_AES_128_CBC_SHA256?
+         */
+        if (first == ECDHE_PSK_BYTE) {
```

Comment:
Should this be wrapped with `#ifndef NO_PSK`?

Reply:
I guess it could be, but none of the other PSK ciphersuites are wrapped in it and I'm not sure why those aren't wrapped.